mne/source_estimate.py

```diff
@@ -383,6 +383,13 @@ def _make_stc(data, vertices, src_type=None, tmin=None, tstep=None,
         src_type = _get_src_type(src=None, vertices=vertices,
                                  warn_text=warn_text)
 
+    if vector and isinstance(data, tuple):
+        # perform all actions on the data kernel only
```

Comment:
imagine yourself in 3years reading this comment. What are your thoughts?

Reply:
Is `# split up data into kernel and sensor data` better? I think it's important to note here that the following computations are performed on the kernel only.

But as a whole, I'm not really satisfied with this part too.
The first problem is that it is misleading that data here represents the imaging kernel and not the actual data (which is stored in sens_data). Then the kernel bool just for this purpose looks ugly.

I thought doing something like this in the beginning:

```python
def _make_stc(data, vertices, ...):

    ...
    rot_target = data[0] if isinstance(data, tuple) else data

    for i, d, n in zip(range(data.shape[0]), rot_target, source_nn):
        data_rot[i] = np.dot(n.T, d)

    if isinstace(data, tuple):
                stc = VectorSourceEstimate((data_rot, sens_data), ...)
```

But i'm not sure whether just reassigning the data array is really a better idea.